backoffice/templates/backoffice/users/users_list_item.html

```diff
@@ -1,8 +1,11 @@
 {% load user_utils %}
 
 <tr>
+  
   <td class="px-6 py-3 text-sm font-medium text-gray-500">
-    {{ u.first_name|title }} {{ u.last_name|title }}
+    <a href="{% url "backoffice_user_with_name" u.username %}" class="underline">
+      {% if u.first_name %}{{ u.first_name|title }} {{ u.last_name|title }}{% else %}{{ u.username }}{% endif %}
```

Comment:
Shouldn't be any users without lastname/firstname, and username is an implementation detail in the system so shouldn't ever be surfaced.